### PR TITLE
Added option for admins to login de-admined

### PR DESCRIPTION
**Pull request by DamianX**
Enable it under preferences -> general settings

### CODE REVIEW

**Comment by DamianX** [comment]
MR 20 on the secret repo is ready

File: __DEFINES/setup.dm [hunk 1]
```diff
@@ -825,6 +825,7 @@ SEE_PIXELS	256
 #define CHAT_GHOSTRADIO 8192
 #define SOUND_STREAMING 16384 // /vg/
 #define CHAT_GHOSTPDA   32768
+#define AUTO_DEADMIN	65536
```
Comment (Exxion) [inline __DEFINES/setup.dm:828]: Ignore this I'm just testing something

**Comment by DamianX** [comment]
I thought this could be buggy because I'm not checking whether we have a holder before deadmining, which is possible in the event someone gets their admin status removed. So I tried to test it but now I can't get the preferences to save at all, not sure what's going on. Fuck it.

**Comment by Intigracy** [comment]
You didn't update preferences_savefile.dm to actually save / load the values

**Comment by DamianX** [comment]
But `toggles` is a bitflags var, which is already saved/loaded?

**Pull request closed without merging**